src/services/fluence-aurora-connector/modules/fluence_aurora_connector/src/latest_block.rs

```diff
@@ -0,0 +1,37 @@
+use marine_rs_sdk::marine;
+
+use crate::curl::send_jsonrpc;
+use crate::jsonrpc::block_number::{BlockNumberReq, BlockNumberResult};
+use crate::jsonrpc::request::check_url;
+
+#[marine]
+pub fn latest_block_number(api_endpoint: String) -> BlockNumberResult {
+    if let Err(err) = check_url(&api_endpoint) {
+        return BlockNumberResult::error(err.to_string());
+    }
+
+    let result = match send_jsonrpc::<_, String>(api_endpoint, BlockNumberReq::new()) {
+        Err(err) => {
+            log::debug!(target: "connector", "request error: {:?}", err);
+            return BlockNumberResult::error(err.to_string());
+        }
+        Ok(result) => result,
+    };
+    log::debug!(target: "connector", "request result: {:?}", result);
+    let result = match result.get_result() {
+        Err(err) => {
+            return BlockNumberResult::error(err.to_string());
+        }
+        Ok(result) => result,
+    };
+
+    let hex_num = result.trim_start_matches("0x");
```

Comment:
mb use here `hex::hex_to_int` here?

Reply:
Also, is this amount of debug log is okay? Mb reduce it?

Reply:
i think it is okay, can disable it any time